warehouse/slave.go

```diff
@@ -135,16 +135,58 @@ func (job *PayloadT) getColumnName(columnName string) string {
 	return warehouseutils.ToProviderCase(job.DestinationType, columnName)
 }
 
+// TODO: work in progress
+func (jobRun *JobRunT) uploadLoadFilesToObjectStorage() ([]int64, error) {
+	job := jobRun.job
+	uploader, err := job.getFileManager()
+	if err != nil {
+		return []int64{}, err
+	}
+	var loadFileIDs []int64
+	var loadFileIDChan chan int64
+	var loadFileErrChan chan error
+	maxParallelLoadsChan := make(chan struct{}, 3) // TODO: add config variable
```

Comment:
Add a config variable.

Reply:
Can be set to a bigger number by default I think